src/tests/ftest/io/nvme_object.yaml

```diff
@@ -0,0 +1,32 @@
+# change host names to your reserved nodes, the
+# required quantity is indicated by the placeholders
+
+hosts:
+ test_servers:
+  - boro-A
+timeout: 2700
+server_config:
+ name: daos_server
+pool:
+ createmode:
+  mode: 511
+ createset:
+  group: daos_server
+ createsize:
+  scm_size: 1000000000
+  size:
+   - 10000000000
+   - 100000000000
+   - 800000000000
+ createsvc:
+  svcn: 1
+container:
+  object_qty: 10
+  record_size:
+    - 1
+    - 50
+    - 100
+    - 1000
```

Comment:
I was looking at log and only 1, 50, 100 record size was taken [Job Log](https://build.hpdd.intel.com/job/daos-stack/job/daos/view/change-requests/job/PR-1065/1/artifact/Functional_Hardware/job-2019-09-07T00.53-8c6fc37-io-nvme_object/job.log)

Reply:
ok